project.md

```diff
@@ -11,41 +11,96 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+Other issues that would create a “suboptimal” output should all be warnings. Examples of this include keywords that cannot be found, non-text data that the program detects, and recursion in the given keyword order. The program should have default behaviors when these characteristics are detected during the validation process. There should also be a constraint on how long a program can run in the case that the DSL missed some edge cases. Warnings should be visible as a list in the GUI, as text within the generated document, or as a comment.
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to coarsely order any text-only document in this language. It should also be easy to create modules of keywords and rules that can be re-used when defining the order of the document.
+
+One possible, but difficult function of this language is to create highly hierarchical orderings in this language. The syntax is intended to be simple as it assumes high-level reordering, but the need necessitates some way of creating nested structures. However, having many nested layers would probably make for messy code in this DSL. This kind of difficulty would likely arise for any code intended to fine-tune choices.
+
+It should be impossible to do recursion, conditionals, or duplication of data in this language. It should also be impossible to generate non-document file types like videos. Additionally, this language should only work with the English language - other languages have different styles of an organization that should alter the many assumptions that this language makes to create the final output.
+
 ### Related work
 
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are DSLs related to this domain. Itinerary creation is used as an example in this Java internal DSL tutorial. This paper shows a DSL-like excerpt of travel itinerary configuration in XCOSEML, which is an extension of the COSEML language (which seems quite DSL-like as well, but I have not spent enough time reading through the documentation to be confident about this claim). These DSLs are proof-of-concepts that a DSL can be created to address the need.
+
+My guess for why there is no DSL specific for this need is because the annoyance is mild enough that people take it for granted as part of the process of organizing documents. Additionally, the more people put up with this annoyance, likely it is that they become better at sorting through vast amounts of information.
+
+COSEML info: https://user.ceng.metu.edu.tr/~coseml/coseml.pdf
+
```

Comment:
I don't think that I know of any tool that operates like your DSL. In the past, I have used mind mapping tools to organize my thoughts but they require me to come in knowing what I want to put where; I think your DSL would be a great tool to make mind mapping more effortless